recipe/meta.yaml

```diff
@@ -13,19 +13,29 @@ source:
     - patches/0001-Add-OSQP_RESPECT_BUILD_SHARED_LIBS-option.patch
     # use libqdldl from conda-forge, build interface in osqp directly
     - patches/0002-build-qdldl-interface-as-part-of-osqp.patch
+      # install osqp_demo and osqp_tester executables to easily make the relocatable,
+      # see: https://github.com/conda-forge/libosqp-feedstock/pull/12#discussion_r1807117589
+    - patches/0003-Install-osqp_demo-and-osqp_tester.patch
 
 build:
-  number: 0
+  number: 1
   run_exports:
     # No ABI policy is documented, let's be conservative
     - {{ pin_subpackage('libosqp', max_pin='x.x.x') }}
+  ignore_run_exports:
+    - numpy
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - cmake
     - ninja
+    # Unittest source code is generated at build time using numpy and scipy
+    - python
+    - numpy
+    - scipy =1.11.4 # as of 0.6.3 imports randn from scipy
```

Comment:
Don't pin to patch version. The constraint should say what it means to do, which is: avoid too-new scipy because it removed a deprecated feature that libosqp still relies on
```suggestion
    # v0.6.3 imports scipy.randn which was removed in 1.12
    - scipy <1.12
```